php-practice_1.php

```diff
@@ -1,41 +1,190 @@
 <?php
 // Q1 変数と文字列
+<?php
+
+$name = "上里 翔星";
+echo "私の名前は「" . $name . "」です。";
 
+?>
 
 // Q2 四則演算
+<?php
 
+$num = 4 * 5;
+
+var_dump($num);
+var_dump($num / 2);
+
+?>
 
 // Q3 日付操作
+<?php
 
+date_default_timezone_set('Asia/Tokyo'); //現在時刻がずれていた為、デフォルト設定
+echo date("現在時刻は、" . "Y/m/d H:i:s" . "です。");
```

Comment:
ダブルクオーテーションを使っているのであれば、文字列結合は不要ではないでしょうか？
```suggestion
echo date("現在時刻は、Y/m/d H:i:sです。");
// または↓
echo '現在時刻は、' . date('Y/m/d H:i:s') . 'です。';
```

Reply:
""を使うなら変数と特殊文字は展開されるので、"現在時刻は、Y/m/d H:i:sです。"と分けずに書いて文字列結合は不要でした。